### PR TITLE
IAC Labcoat

### PR DESCRIPTION
**Pull request by WilliamMurdoch**
Adds a labcoat in the IAC colors.

### CODE REVIEW

**Comment by WilliamMurdoch** [comment]
!review

File: code/modules/client/preference_setup/loadout/loadout_suit.dm [hunk 1]
```diff
@@ -109,6 +109,14 @@
 	allowed_roles = list("Chief Medical Officer", "Physician", "Surgeon", "Pharmacist", "First Responder", "Medical Intern")
 	flags = GEAR_HAS_DESC_SELECTION
 
+/datum/gear/suit/iaclabcoat
+	display_name = "IAC labcoat"
+	description = "It's a standard medical labcoat designed to be worn by the Instellar Aid Corps."
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_suit.dm:114]:
```suggestion
	description = "It's a standard medical labcoat designed to be worn by the Interstellar Aid Corps."
```

File: code/modules/clothing/suits/labcoat.dm [hunk 2]
```diff
@@ -99,3 +99,10 @@
 	desc = "A suit that protects against minor chemical spills. Offers slightly more protection against biohazards than the standard model. Comes in Hephaestus colours."
 	icon_state = "labcoat_heph"
 	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 60, rad = 0)
+
+/obj/item/clothing/suit/storage/toggle/labcoat/iac
+	name = "iac labcoat"
+	desc = "A suit that protects against minor chemical spills. Offers slightly more protection against biohazards than the standard model. Comes in IAC colors."
+	icon_state = "labcoat_iac"
+	item_state = "labcoat_iac"
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 60, rad = 0)
```
Comment (NonQueueingMatt) [inline code/modules/clothing/suits/labcoat.dm:108]: Update this branch and use the armor defines.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Comment by WilliamMurdoch** [comment]
Alright, I messed up when I was pushing the changes. I'm going to close this redo this in a way that won't mess up.

**Pull request closed without merging**